easybuild/easyconfigs/p/PLYfile/PLYfile-0.7-foss-2019a.eb

```diff
@@ -0,0 +1,33 @@
+easyblock = 'PythonPackage'
+
+name = 'PLYfile'
```

Comment:
should be just `plyfile`, based on https://github.com/dranjan/python-plyfile#installing-plyfile?